xml/System/Convert.xml

```diff
@@ -18626,13 +18633,17 @@
         <Parameter Name="options" Type="System.Base64FormattingOptions" Index="3" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="bytes">To be added.</param>
-        <param name="chars">To be added.</param>
-        <param name="charsWritten">To be added.</param>
-        <param name="options">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="bytes">A read-only span of 8-bit unsigned integers.</param>
+        <param name="chars">When this method returns <see langword="true" />, a span containing the string representation in base 64 of the elements in <paramref name="bytes" />. If the length of <paramref name="bytes" /> is 0, or when this method returns <paramref name="false" />, nothing is written into this parameter.</param>
```

Comment:
Does this method returns true if the length of bytes is 0? if not, we could simplify that sentence.

"nothing is written into this parameter" -> wouldn't it be better to say the span is uninitialized or something?